pkg/deploy/rp.go

```diff
@@ -883,7 +927,8 @@ func (g *generator) template() *arm.Template {
 		t.Resources = append(t.Resources, g.pip(), g.lb(), g.vmss())
 	}
 	// clustersKeyvault must preceed serviceKeyvault due to terrible bytes.Replace below
-	t.Resources = append(t.Resources, g.zone(), g.clustersKeyvault(), g.serviceKeyvault(), g.vnet())
+	t.Resources = append(t.Resources, g.zone(), g.clustersKeyvault(), g.serviceKeyvault(), g.rpvnet(), g.pevnet())
```

Comment:
nit: this could be done for readability:
```
	t.Resources = append(t.Resources, g.zone(), g.clustersKeyvault(),
		g.serviceKeyvault(), g.rpvnet(), g.pevnet(),
		g.halfPeering("rp-vnet", "rp-pe-vnet-001"),
		g.halfPeering("rp-pe-vnet-001", "rp-vnet"))
```

Reply:
append call split into several lines.